src/main/java/org/folio/circulation/support/http/server/JsonHttpResponse.java

```diff
@@ -33,6 +33,10 @@ public static HttpResponse unprocessableEntity(JsonObject body) {
     return new JsonHttpResponse(422, body, null);
   }
 
+  public static HttpResponse noContent() {
+    return new JsonHttpResponse(204, null, null);
+  }
```

Comment:
If there is no content, why would this be a JSON response?

Reply:
Fixed